documentation/modules/operators/ref-operator-cluster.adoc

```diff
@@ -303,6 +308,32 @@ Add `-ServiceAccountPatching` to the `STRIMZI_FEATURE_GATES` environment variabl
 
 NOTE: The `ServiceAccountPatching` feature gate moved to beta stage in Strimzi 0.27.0 and is expected to remain in the beta stage until Strimzi 0.30.
 
+[id='ref-operator-use-strimzi-pod-sets-feature-gate-{context}']
+=== Use `StrimziPodSets` feature gate
+
+Currently, Strimzi relies on StatefulSets to create and manage Pods for the ZooKeeper and Kafka clusters.
```

Comment:
If we're not referring directly to the object/resource with backticks (`Pod`) we can use lower case like pods, nodes etc. StatefulSets could be split to be _stateful sets_.